docs/reference/ilm/getting-started-ilm.asciidoc

```diff
@@ -3,77 +3,87 @@
 [[getting-started-index-lifecycle-management]]
 == Getting started with {ilm}
 
-Let's jump into {ilm} ({ilm-init}) by working through a hands-on scenario.
-This section will leverage many new concepts unique to {ilm-init} that
-you may not be familiar with. The following sections will explore
-these in more details.
-
-The goal of this example is to set up a set of indices that will encapsulate
-the data from a time series data source. We can imagine there is a system
-like {filebeat-ref}[Filebeat] that continuously indexes documents into
-our writing index. We wish to roll over the index after it reaches a size
-of 50 gigabytes, or has been created 30 days ago, and then delete the index
-after 90 days.
+This tutorial demonstrates how to use {ilm} ({ilm-init}) 
+to manage indices that contain time-series data. 
+
+When you use a mechanism like Filebeat to continuously index documents into {es},
+you typically use an index alias so you can periodically roll over to a new index.
+This enables you to implement a hot-warm-cold architecture to meet your performance
+requirements for your newest data, control costs over time, enforce retention policies,
+and still get the most out of your data.
```

Comment:
can we emphasize that any data source is ok to be used? (I think the current version, albeit more composed, ties ILM to Filebeat, which is something that some users might not be familiar with) Maybe something along the lines of
"When using a a time series data source like Filebeat to continuously ..." ?

Reply:
Changed to, "When you continuously index timestamped documents into {es} using 
Filebeat, Logstash, or some other mechanism..."